doc/cdi-operator-olm.md

```diff
@@ -0,0 +1,26 @@
+# OLM intergartion
+##OLM Overview
+https://github.com/kubevirt/kubevirt/blob/master/docs/devel/olm-integration.md
+
+
+## CDI OLM manifests 
+1. in order to obtain olm bundle manifest run the following command
+```bash
+DOCKER_REPO=<repo> DOCKER_TAG=<docker tag> PULL_POLICY=<pull policy> VERBOSITY=<verbosity> CSV_VERSION=<CSV version> QUAY_NAMESPACE=<namespace> QUAY_REPOSITORY=<application name> make manifests
+```
```

Comment:
I wonder how bad is to add a real example so if one who does not understand exactly how this works, will get better idea when looking at the values of the parameters above. i.e CSV version. A note can be added to say that that this is only example, if we are afraid that it can be misleading.

Reply:
okay. I see you have examples in hack/README.md, so please ignore previous comment.